### DISCUSSION TITLE
Labels for unmatched packages

### DISCUSSION THREAD

**Discussion by jalaziz** [renovatebot/renovate#19137] [Request Help] [answered]
I apologize if this has been asked before, but a quick search didn't reveal anything obvious.

We have a monorepo and dependencies are managed by various teams based on the affected component. When a dependency update is discovered for a component that has not been correct labeled/categorized, we'd like to add a PR label indicating as much.

Using the top-level `label` configuration doesn't do the trick unless each `packageRule` overrides the top-level `label` completely. Also, as far as I can tell, there is no "fallback" `packageRule`.

Fundamentally, what we're looking for is a way to say "if no package rules match, add these labels". Is it possible?

**Comment by viceice** [comment, marked as answer]
no

**Reply by jalaziz** [reply]
Thanks for the quick reply! We'll try to re-think our config then.
